websites/C/Club Penguin Journey/presence.ts

```diff
@@ -0,0 +1,38 @@
+const presence = new Presence({
+		clientId: "620721262112538625", // CLIENT ID FOR YOUR PRESENCE
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/2MUm0oA.jpeg",
+		startTimestamp: browsingTimestamp,
+	};
+
+	switch (document.location.hostname) {
+		case "cpjourney.net": {
+			if (document.location.pathname === "/") {
+				presenceData.details = "Waddling around";
+				presenceData.state = "Viewing Home";
+			} else if (document.location.pathname.startsWith("/news/")) {
+				presenceData.details = "Reading post:";
+				presenceData.state =
+					document.querySelector(".post_title__MNhEJ").textContent;
```

Comment:
Try using selectors which avoid computer generated class names. For example, `[class*=post_title]`

Reply:
They all used the same class "post_title__MNhEJ" so I thought it would've been fine, my bad. Let me change it.